src/Uno.UI/ts/Windows/Storage/ApplicationDataContainer.ts

```diff
@@ -36,11 +41,14 @@ namespace Windows.Storage {
 		 * Set a value to localStorage
 		 * */
 		private static setValue(pParams: number): boolean {
-			const params = ApplicationDataContainer_SetValueParams.unmarshal(pParams);
+			try {
+				const params = ApplicationDataContainer_SetValueParams.unmarshal(pParams);
 
-			const storageKey = ApplicationDataContainer.buildStorageKey(params.Locality, params.Key);
+				const storageKey = ApplicationDataContainer.buildStorageKey(params.Locality, params.Key);
 
-			localStorage.setItem(storageKey, params.Value);
+				localStorage.setItem(storageKey, params.Value);
+			} catch (e) {
+			}
```

Comment:
It feels wrong to have an empty catch statement like this. It's not clear what to do there, but I think we should log something somewhere...

Reply:
@carldebilly I think this mostly have the exact same semantics as the first commit if the PR?
I have no idea if there is a specific logging pattern we follow in TS code.

Reply:
Logging that support for local storage is needed, whether it's done once or every time.

Reply:
@jeromelaban I mean, regular `console.log(..)` or is there something similar to `this.Log()` that we use in C# code?

Reply:
`console.debug` would do.